docs/articles/posts/20241129_microsoft_hack.md

```diff
@@ -0,0 +1,52 @@
+description: >
+    The NHS England Data Science team, as well as a range of other analysts from across the organisation, attended an AI Hackathon at Microsoft, organised by the Data Science Team together with Microsoft and Kainos, with the key stakeholders being the NHS Websites Team. 
+---
+
+> The NHS England Data Science team, as well as a range of other analysts from across the organisation, attended an AI Hackathon at Microsoft, organised by the Data Science Team together with Microsoft and Kainos, with the key stakeholders being the NHS Websites Team. In this article, the author shares her experiences at the event. 
+
+<!-- more -->
+
+Months ago, when I was invited to this hackathon, I sat at my desk and reluctantly signed up because everyone else was attending, sighing to myself about "please not another hackathon", and this was the energy I brought with me to the beautiful Storey Club, in Paddington, London, last week Tuesday. I thought "well at least I'll get a free lunch", but I was proven so wrong in my pessimism. Every team had either a Kainos or Microsoft representative, as well as a range of participants from across NHS England, including at least a few Data Science Team members per team. And unlike a usual hackathon, this one had stakeholders and well defined use cases, we were all working to one common goal. 
+
+The use cases were outlined as: 
+
+1. **Search & Summarise**: using an extract of the NHS Corpus developed by one of our PhD interns, Sam Hollands, (we used two websites worth of data, out of 600 NHS websites, and it was already 25GB worth of data!), we had to develop an LLM that used RAG to sift through that data and retrieve answers to queries based on these NHS websites.
+2. **Explaining to a 12 year old**: Adapting the search and summarise for different personas, one of which being a 12 year old child
+3. **Duplication & Conflict Detection**: comparing the two sites and identifying duplicate information and conflicting information.
+
+The aim was for the websites team to have work to improve upon and be able to implement them in their websites. Having use cases meant that it was so much easier to split work up, and get our heads down, with no one in our teams finding themselves with nothing to do. It also meant we had a structured approach, making the experience so much more fulfilling. However, even with this, I was expecting that when it came to presenting, we'd end up watching 9 identical presentations, that just got slightly further along the process depending on the team. Once again, I was proved wrong by the creativity and range of approaches that the different teams took. We ranged from teams that had taken a thin slice approach to all three usecases, to teams that had gone really in depth in just one of them, to teams that had taken use case number 3 and made it into a tool that could be used by the websites team to improve the websites on the backend, removing conflict and duplication. 
```

Comment:
"with no one in our teams finding themselves with nothing to do" - instead: everyone in our teams finding themselves with something to do

Reply:
Maybe remove this "However, even with this, I was expecting that when it came to presenting, we'd end up watching 9 identical presentations, that just got slightly further along the process depending on the team. Once again, " . Change the next sentence to "I was delighted by the creativity..."